src/golive/cdn.md

```diff
@@ -52,3 +52,51 @@ When using a CDN the Platform.sh router's HTTP cache becomes redundant.  In most
         # Disable the HTTP cache on this route. It will be handled by the CDN instead.
         enabled: false
 ```
+
+## Secure/hide your project from being accessed directly
+
+When using a CDN, you generally don't want users to access your platform.sh project directly. Luckily, you can secure your connection between you and the CDN with [client authenticated TLS](/configuration/routes/https.html#client-authenticated-tls). 
+
```

Comment:
I think we also need to point out something like: Alternatively, you can restrict origin access to the CDN provider and your development team by using IP based restrictions <link to documentation>.

Reply:
Or using password auth, as said by Florian.